### ISSUE TITLE
Websocket errors found in logs

### ISSUE THREAD

**Issue by koldogut** [StyraHem/ShellyForHASS#635]
Checking the logs I've found lots of entires pointing to websocket errors

```
Logger: websocket
Source: /usr/local/lib/python3.9/site-packages/websocket/_logging.py:61
First occurred: 4:28:52 PM (93 occurrences)
Last logged: 5:59:00 PM

error from callback <bound method WebSocket.on_open of <pyShelly.ws_client.WebSocket object at 0x14dc82abf6d0>>: on_open() missing 1 required positional argument: 'ws'
```

Any idea of the reasons? I'm afraid it could cause major issues if not fixed. Please find below versions and additional info:

ShellyForHass (Shelly integration) v 0.3.3
Version  core-2021.12.4
Installation Type Home Assistant Container
Python Version 3.9.7

**Comment by hakana** [comment]
Yes, I notice that also when I enabled restricted login on a device. I working on a fix for that right now.

I guess you have restricted login also on a Plus/Pro device?

**Comment by koldogut** [comment]
It's indeed a ShellyPro4PM but the thing is I that set a password first but it is not enabled anymore and the error in the logs persists. If I could provide you with any log or further checks just let me know. 
(great work by the way!)
